src/components/markdown/PaperPage.tsx

```diff
@@ -46,7 +56,7 @@ export const MarkdownContents: React.FunctionComponent<{
             ← Previous: {posts[prev].shortTitle || posts[prev].title}
           </PageLink>
         ) : (
-          <Link to="./">← Table of Contents</Link>
+          <Link to="./">← {rootTitle}</Link>
```

Comment:
Could you document this parameter in the function declaration? It's not obvious to me what "rootTitle" means or where it would appear. Alternatively, find a self-documenting parameter name.

Reply:
rseolved. good call.